vocode/streaming/synthesizer/rime_synthesizer.py

```diff
@@ -52,13 +51,14 @@ async def create_speech(
         body = {
             "text": message.text,
             "speaker": self.speaker,
+            "samplingRate": self.samplingRate
```

Comment:
can you pass in `None` for both `speaker` and `samplingRate`?

Reply:
We don't do any checking on our end but you'll get a 400 if you don't send valid `speaker`

Reply:
how do you mean @ajar98 can you give some context? thanks